distribution/client/src/lib/downloader.js

```diff
@@ -27,20 +27,38 @@ class Downloader {
 
     logger.info('Fetching %s and saving to %s', item, filename);
 
-    return new Promise((resolve, reject) => {
-      fetch(item).then((res) => {
-        const output = fs.createWriteStream(filename);
+    let options = {
+      uri: item,
+      verboseLogging: true,
+      method: 'GET',
+      headers: {
+        'User-Agent': 'evergreen-client'
+      },
+      simple: true,
+      resolveWithFullResponse: true,
+      encoding: null,
+      timeout: 10 * 1000,
+      retry: 3 // make it configurable?
```

Comment:
There's no need for end-user configuration of something like this, we're trying to get away from the infinite-configuration-switches remember :smiling_imp: